app/interactors/content/update_interactor.rb

```diff
@@ -52,15 +57,13 @@ def update_preview
     FailsafeDraftPreviewService.call(edition)
   end
 
-  def update_params(edition)
-    contents_params = edition.document_type.contents.map(&:id)
+  def change_note_params
+    params.require(:revision).permit(:update_type, :change_note)
+  end
 
-    params.require(:revision)
-      .permit(:update_type, :change_note, :title, :summary, contents: contents_params)
-      .tap do |p|
-        p[:title] = p[:title]&.strip
-        p[:summary] = p[:summary]&.strip
-        p[:base_path] = GenerateBasePathService.call(edition.document, p[:title])
-      end
+  def update_params
```

Comment:
I wasn't concerned about it, and now I see that it's different to the original interpretation, since there are two such methods. On the other hand, I also look how it distinguishes `change_note`, which is tangential to the purpose of the page. But `content_params` sounds better overall. Updated.

Reply:
Ah seems we've focused on different words in the compound. I wasn't fazed by the update word but by the params word based on it implying it's an ActionController::Parameters object.

Reply:
Oh dear ‼️ . I can't say I'm concerned about the use of `_params`, as they are parameters (technically, 'arguments'). Other words like `_attributes` and `_data` spring to mind, but it seems overkill to expend more effort on an internal method.